### ISSUE TITLE
There is a input sequence, how to calculate probability for the specail output sequence

### ISSUE THREAD

**Issue by yick2232** [Kyubyong/transformer#7]

**Comment by RayXu14** [comment]
notice logits in Graph. you can softmax logits, then you get probability
